invenio_rdm_records/resources/serializers/bibtex/schema.py

```diff
@@ -268,15 +269,16 @@ def _format_output_row(self, field, value):
                 out = "  {0:<12} = {{{1}}},\n".format(field, value)
         return out
 
-    def _get_citation_key(self, data):
+    def _get_citation_key(self, data, original_data):
         """Return citation key."""
         id = data["id"]
 
-        creator = data.get("creator", None)
-        if creator is None:
+        creators = original_data["metadata"].get("creators", [])
+        if not creators:
             return id
 
-        name = creator.get("name")
+        creator = creators[0].get("person_or_org", None)
```

Comment:
```suggestion
        creator = creators[0].get("person_or_org", {})
```

Otherwise `creator.get()` will throw an exception.